### ISSUE TITLE
Currently playing dont refresh

### ISSUE THREAD

**Issue by fczuardi** [lnbits/jukebox#12]
If the user leaves the music selection page opened, the "Currently playing" song doesnt updates after the music ends.

Steps to reproduce:

- open the jukebox page
- pay for selecting the next track
- wait for the current track to end

Expected behavior:

- the next track in the queue (probably yours) starts playing
- the "current playing" title and album cover gets updated

**Comment by fczuardi** [comment]
possible fix:

```diff
diff --git a/templates/jukebox/jukebox.html b/templates/jukebox/jukebox.html
index 15e551e..bcad502 100644
--- a/templates/jukebox/jukebox.html
+++ b/templates/jukebox/jukebox.html
@@ -258,6 +258,7 @@
     },
     created() {
       this.getCurrent()
+      window.setInterval(this.getCurrent, 10 * 1000)
       this.playlists = JSON.parse('{{ playlists | tojson }}')
       this.selectedWallet.inkey = '{{ inkey }}'
       self = this
```

**Comment by talvasconcelos** [comment]
Also fixed in #16
